app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, file, file_copy = command.split()
+    if cp != "cp" and file == file_copy:
```

Comment:
Ти нічого не робиш, якщо в тебе неправильна команда І файли збігаються має бути 
```suggestion
    if cp == "cp" and file != file_copy:
         # do the copying
```